mlflow/sklearn/utils.py

```diff
@@ -58,14 +66,13 @@ def _get_Xy(args, kwargs, X_var_name, y_var_name):
     return kwargs[X_var_name], kwargs[y_var_name]
 
 
-def _get_samples_labels_and_predictions(fitted_estimator, fit_args, fit_kwargs, fit_arg_names):
+def _get_samples_and_labels(fit_args, fit_kwargs, fit_arg_names):
     # In most cases, X_var_name and y_var_name become "X" and "y", respectively.
     # However, certain sklearn models use different variable names for X and y.
     X_var_name, y_var_name = fit_arg_names[:2]
     X, y_true = _get_Xy(fit_args, fit_kwargs, X_var_name, y_var_name)
-    y_pred = fitted_estimator.predict(X)
 
-    return X, y_true, y_pred
```

Comment:
What was the motivation for this change? In every instance where we call `_get_samples_and_labels`, the subsequent line calls `y_pred = fitted_estimator.predict(X)`, which is effectively the same as keeping that logic in this function.